src/assets/main.css

```diff
@@ -61,7 +61,6 @@
   * @import "utilities/skew-transforms";
   */
 
-html,
```

Comment:
Why did you have this? This was causing the body not to take up 100% of the height

Reply:
Its needed, removing it will break the login and register pages height